src/components/ToggleSwitch.js

```diff
@@ -37,20 +41,93 @@ export default class ToggleSwitch extends React.Component<ToggleSwitchDefaultPro
   offButton: ?HTMLDivElement;
 
   render() {
-    const disabledClass = this.props.disabled ? 'disabled' : '';
+    const {
+      disabled,
+      noLabel,
+      offLabel,
+      on,
+      onChange,
+      onLabel,
+    } = this.props;
+
+    const disabledClass = disabled ? 'disabled' : '';
     const containerClass = `toggle-switch-container ${disabledClass}`;
-    const onClass = `toggle-switch toggle-switch-on ${this.props.on ? 'selected' : ''} ${disabledClass}`;
-    const offClass = `toggle-switch toggle-switch-off ${this.props.on ? '' : 'selected'} ${disabledClass}`;
+    const onClass = `toggle-switch toggle-switch-on ${on ? 'selected' : ''} ${disabledClass}`;
+    const offClass = `toggle-switch toggle-switch-off ${on ? '' : 'selected'} ${disabledClass}`;
+
+    if (noLabel) {
+      const buttonStyle = on && !disabled
+        ? {
+          background: '#2F75B0',
```

Comment:
Hard-coding color-codes like this is something we avoid in SUIT.
This toggle is the color of the default `@brand-primary` less variable in the `variables.less` file, but if somebody in Solutions or Professional Services creates a branded UI and changes this variable to suit a client or prospect, this toggle's color will not change with it.
I realize this isn't the ideal fix, but until we refactor the styling mechanism we should find existing class names which apply these colors and use those instead of calling the colors out directly by hash code.

Reply:
That makes sense, is there a specific file where all color variables are defined for suit?